Woof/Woof/View/OwnerMainTabView.swift

```diff
@@ -0,0 +1,72 @@
+import SwiftUI
+
+struct OwnerMainTabView: View {
+    @State private var selection: Tab = .sitters
+
+    init() {
+        customizeTabBar()
+    }
+
+    var body: some View {
+        NavigationView {
+            TabView(selection: $selection) {
+                Group {
+                    Text("List of pet sitters")
+                        .tabItem {
+                            Label("Sitters", systemImage: "magnifyingglass")
+                        }
+                        .tag(Tab.sitters)
+
+                    Text("Walkings stub")
+                        .tabItem {
+                            Label("Walkings", systemImage: "pawprint")
+                        }
+                        .tag(Tab.walkings)
+
+                    Text("Pet owner info")
+                        .tabItem {
+                            Label("Profile", systemImage: "person.text.rectangle")
+                        }
+                        .tag(Tab.profile)
+                }
+            }
+            .navigationTitle(selection.header)
+            .navigationBarTitleDisplayMode(.inline)
+        }
+    }
+
+    private func customizeTabBar() {
+        let tabBarAppearance = UITabBar.appearance()
+        tabBarAppearance.unselectedItemTintColor = UIColor(Color.grayDark)
+    }
+}
+
+struct OwnerMainTabView_Previews: PreviewProvider {
+    static var previews: some View {
+        OwnerMainTabView()
+    }
+}
+
+extension OwnerMainTabView {
```

Comment:
Should we put this extension and/or Tab into the separate file?

Reply:
Move to a separate file to improve code readability.